tests/Unit/NumericalAlgorithms/SphericalHarmonics/Test_Spherepack.cpp

```diff
@@ -21,6 +21,521 @@
 namespace ylm {
 namespace {
 
+// [spectre_cce_grid_point_locations]
+struct SpectreCceGridPointLocations {
```

Comment:
[optional] To avoid so many NOLINTs, you could wrap this whole part in

```cpp
// NOLINTBEGIN
...
// NOLINTEND
```

Reply:
This is a great idea, thanks!